src/main/java/uk/ac/ebi/embl/api/translation/CdsTranslator.java

```diff
@@ -439,11 +439,11 @@ public ValidationResult configureFromFeature(
     // then the left and right partiality are reversed between CdsFeature
     // and Translator.
     if (compoundLocation.isComplement()) {
-      translator.setLeftPartial(compoundLocation.isRightPartial());
-      translator.setRightPartial(compoundLocation.isLeftPartial());
+      translator.setLeftPartial(compoundLocation.isThreePrime());
+      translator.setRightPartial(compoundLocation.isFivePrime());
     } else {
-      translator.setLeftPartial(compoundLocation.isLeftPartial());
-      translator.setRightPartial(compoundLocation.isRightPartial());
+      translator.setLeftPartial(compoundLocation.isFivePrime());
+      translator.setRightPartial(compoundLocation.isThreePrime());
```

Comment:
Potentially unsafe. Best would be to review change Translation left and right partiality as well.